cloudslang-cli/pom.xml

```diff
@@ -332,8 +338,125 @@
+                                <goals>
+                                    <goal>checkout</goal>
+                                </goals>
+                                <configuration>
+                                    <connectionUrl>scm:git:git@github.com:CloudSlang/cs-content.git</connectionUrl>
+                                    <checkoutDirectory>${project.build.directory}/cs-content</checkoutDirectory>
+                                    <scmVersion>${cs-content.revision}</scmVersion>
+                                    <scmVersionType>${cs-content.type}</scmVersionType>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-resources-plugin</artifactId>
+                        <version>3.0.2</version>
+                        <executions>
+                            <execution>
+                                <id>copy-content</id>
+                                <phase>prepare-package</phase>
+                                <goals>
+                                    <goal>copy-resources</goal>
+                                </goals>
+                                <configuration>
+                                    <outputDirectory>${project.build.directory}/cslang-cli</outputDirectory>
+                                    <resources>
+                                        <resource>
+                                            <directory>${project.build.directory}/cs-content</directory>
+                                            <filtering>false</filtering>
+                                            <includes>
+                                                <include>**</include>
+                                            </includes>
+                                            <excludes>
+                                                <exclude>**/ci-env/**</exclude>
+                                                <exclude>**/test/**</exclude>
+                                                <exclude>*.yml</exclude>
+                                                <exclude>CONTRIBUTING.md</exclude>
+                                                <exclude>README.md</exclude>
+                                                <exclude>**/configuration/runconfig/**</exclude>
+                                            </excludes>
+                                        </resource>
+                                    </resources>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-assembly-plugin</artifactId>
```

Comment:
what about the default libs in `/maven/repo/ folder? What will be the decision?

Reply:
It would be nice to have them all downloaded and the content ready when first running the CLI without having to download any other resources.

Reply:
agree - please add the basic `@action` dependencies - please also investigate the size of CLI zip after these changes